recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ outputs:
     version: {{ version }}
     build:
       noarch: python
+      skip: true  # [not linux]
```

Comment:
I think you mean to skip the `conda-store` output, so it's only built once on Linux, since that's true noarch and we only need one copy.

The `-server` variant is the one that needs two builds: Linux and macOS.